tests/extendrtests/R/extendrtests.R

```diff
@@ -1,7 +1,44 @@
-#' Integration tests for calling extendr from an R package.
-#'
-#' Integration tests for calling extendr from an R package.
 #' @name extendrtests
 #' @docType package
 #' @useDynLib extendrtests, .registration = TRUE
-NULL
\ No newline at end of file
+NULL
+
+# Generated by
+#
+# cat(.Call("wrap__make_extendrtests_wrappers", use_symbols=TRUE))
+
+# '@export
```

Comment:
Notice that the `'` is in the wrong location. You have a space between `#` and `'`, `# '`, instead of `#' `. This means the `@export` directive is ignored. The tests pass because testthat has access to unexported functions also.

Reply:
Just as I thought it was all working!